plone/namedfile/adapters.py

```diff
@@ -127,3 +127,6 @@ def get_original_image_url(self, fieldname, width, height):
         )
         # Corrupt images may not have a scale.
         return scale.url if scale else None
+
+    def _scale_view_from_url(self, url):
+        return url.replace(self.context.absolute_url(), "").lstrip("/")
```

Comment:
https://github.com/zopefoundation/Zope/blob/master/src/ZPublisher/HTTPRequest.py#L245

see https://community.plone.org/t/image-scale-url-pointing-to-local-url-in-classic-ui/15390/8